### ISSUE TITLE
Warn only once about not finding the `XDG_*` environment variables

### ISSUE THREAD

**Issue by cipriancraciun** [lawl/NoiseTorch#32]
At the moment, if one doesn't have `XDG_*` environment variables configured, the `NoiseTorch` daemon complains about that fact each time it wants to use them;  for example the log is full of lines like:
~~~~
Couldn't resolve $XDG_CONFIG_HOME falling back to '...'
~~~~

It would be nice if the tool would complain only once about this when it starts, and be silent afterwards.

**Comment by lawl** [comment]
Agree, though these aren't currently resolved at startup time and I'd like to avoid introducing some state there just to cull a couple of log lines. I'll leave this open as a low priority enhancement for now, as it's just a slight cosmetic issue in the log.

**Comment by Calinou** [comment]
Does it really need to warn about XDG environment variables not being found? Most people don't override those, which mean the default paths are used instead. I don't see how this is an error (or even a warning).

**Comment by lawl** [comment]
It's not a warning, it's debug output for me so I can see whats going on when people report issues.

There's an argument to hide certain things behind a `verbose` flag, but the log isn't cluttered enough yet imo that I think this is really an issue at the moment.

**Comment by lawl** [comment]
i think this can be closed now, since we don't log to stdout anymore and there should be no user expectation for debug logs to not contain debug output.
